common/reactive-grpc-common/src/main/java/com/salesforce/reactivegrpc/common/AbstractSubscriberAndProducer.java

```diff
@@ -223,7 +218,7 @@ void drain() {
         }
 
 
-        for (;;) {
+        do {
```

Comment:
Let's keep it as in the original code. From my point of view it preserves readability

Reply:
Fixed